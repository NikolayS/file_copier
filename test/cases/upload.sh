#!/bin/bash

# to run this scenario, COPIERHOST should be defined (`export COPIERHOST=http://your.copier.host.name`)

path=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
http -hf POST "http://$COPIERHOST/?wh=1&md5=1" fileRaw@"$path/upload.sample.png" | grep -v "Date:"

