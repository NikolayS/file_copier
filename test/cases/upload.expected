HTTP/1.1 200 OK
Server: nginx
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
X-File-Copier-Md5: 19c7b466a3a68bb2b4de455e42f7b540
X-File-Copier-Img-Width: 230
X-File-Copier-Img-Height: 80
X-File-Copier-Size: 3040
X-Location: /data/1e/88/2c/7d/1e882c7d7b0edc400905b5a3286f1fe36ea4ac9724cece9cb07fc787c5a4d7de.png
Content-Encoding: gzip

